Makefile

```diff
@@ -46,12 +47,21 @@ test-e2e-debug: build $(src) $(tests)
 	DEBUG=1 $(wdio) wdio.conf.js
 test-e2e-ci: build $(src) $(tests)
 	@$(wdio) wdio.ci.conf.js
+test-tsc: tsc
+	@$(tsc)
+	@rm -rf ./dist
 
 lint: build
 	@$(eslint)
 lint-fix: build
 	@$(eslint) --fix
 
+tsc:
+	@$(tsc) -p types -outFile ./build/index.d.ts
+
+tsc-watch:
+	@$(tsc) -w -p types -outFile ./build/index.d.ts
```

Comment:
These new targets will need to be added to `.PHONY`

Reply:
let's change the output to `./build/recurly.d.ts` to match the rest of the lib's output